main.py

```diff
@@ -103,18 +108,52 @@ def main():
     parser.add_argument(
         "--calibration_standard",
         required=False,
+        default="fames",
         help="Calibration standard to use for the GCMS metadata generation. Must be a value from the NMDC Schema. Default is 'fames'.",
     )
     parser.add_argument(
         "--configuration_file",
         required=False,
+        default="emsl_gcms_corems_params.toml",
```

Comment:
I do not think this should have a default actually.  Also unrelated to this PR though.

Reply:
Also on the help - this should be the corems configuration for all the workflows (not just GCMS).